finance_manager/src/main/java/com/exadel/finance/manager/mapstruct/mappers/MapStructMapper.java

```diff
@@ -0,0 +1,25 @@
+package com.exadel.finance.manager.mapstruct.mappers;
+
+import com.exadel.finance.manager.wallet.model.entity.Wallet;
+import com.sandbox.model.WalletRequestDto;
+import com.sandbox.model.WalletResponseDto;
+import java.util.List;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+
+@Mapper(componentModel = "spring")
+public interface MapStructMapper {
+    @Mapping(target = "_default", source = "isDefault")
+    WalletRequestDto walletToWalletRequestDto(Wallet wallet);
+
+    @Mapping(target = "_default", source = "isDefault")
+    WalletResponseDto walletToWalletResponseDto(Wallet wallet);
+
+    @Mapping(target = "isDefault", source = "default")
+    Wallet walletResponseDtoToWallet(WalletResponseDto walletResponseDto);
```

Comment:
I believe flow is 
RequestDTO -> Entity -> ResponseDTO

Do we need to map "response to entity" (walletResponseDtoToWallet) and "entity to request" (walletToWalletRequestDto) ?

Reply:
Yes, in the last commit I sort of deleted unnecessary mappings